src/components/Loader/README.md

```diff
@@ -0,0 +1,26 @@
+Loader
+---------------
+
+Component for rendering loading state (flashing bars)
+
+## Properties
+| Property     | Type       | Required | Default | Description |
+|:---          |:---        |:---:     |:---     |:---         |
+| size         | `String`   |          | 'm'     |Component size. Available values: `s`, `m`, `l` |
+| className    | `String`   |          |         |CSS class   |
```

Comment:
Let's rewrite this into TS interface with JSDoc comments. See Button.md for reference